backend/decky_loader/main.py

```diff
@@ -207,6 +207,7 @@ async def inject_javascript(self, tab: Tab, first: bool=False, request: Request|
                 await tab.close_websocket()
                 self.js_ctx_tab = None
                 await restart_webhelper()
+                await sleep(5)
```

Comment:
What does this sleep do? May be worth adding a comment so in the future we still know what this does

Reply:
done